File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Product/ProductOptionTemplateType.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.ClientCare.Contracts.Entities.Product
{
    public class ProductOptionTemplateType
    {
        public int ProductOptionTemplateTypeId { get; set; }
        public int? ProductOptionId { get; set; }
        public TemplateTypeEnum? TemplateType { get; set; }
        public int? TemplateId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/product-manager/models/product-option-template-type.ts
import { TemplateTypeEnum } from 'projects/shared-models-lib/src/lib/enums/template-type-enum';

export class ProductOptionTemplateType {
  productOptionTemplateTypeId: number;
  productOptionId: number;
  templateType: TemplateTypeEnum;
  templateId: number;
}
